src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -616,9 +632,7 @@ export default {
 			}
 		},
 
-		async createConversation(name) {
-			const response = await createPrivateConversation(name)
-			const conversation = response.data.ocs.data
+		addAndOpenConversation(conversation) {
```

Comment:
Conversation was already added on backend, so it's just a shortcut for putting it to LeftSidebar and open. Maybe, rename like this?
```suggestion
		openConversation(conversation) {
```
or
```suggestion
		switchToConversation(conversation) {
```

Apart from that, nice reuse